### ISSUE TITLE
Create conversion script for JSON -> CSV

### ISSUE THREAD

**Issue by codingo** [disclose/disclose#102]
The file https://github.com/disclose/disclose/blob/master/program-list/program-list.csv has drifted from the associated json file.

This pull request is to track work to create a python script to handle the building of `program-list.csv` from the json file so contributors only need update a single source of truth.

**Comment by codingo** [comment]
Minimal work needed to finish, WIP:

```
import json
import sys
import argparse
import csv
parser = argparse.ArgumentParser()
parser.add_argument('-j', '--jsonfile', help='Input json file', type=str)
parser.add_argument('-o', '--outfile', help='Output csv file name', type=str)
args = parser.parse_args()
if(args.jsonfile is None or args.outfile is None):
    print(" error: input json file and outfile is required")
    sys.exit(0)
with open(args.jsonfile, 'r') as f:
    programs = json.load(f)
with open(args.outfile, 'w', newline='') as file:
    writer = csv.writer(file)
    writer.writerow(["program_name", "policy_url"])
    for program in programs:
        writer.writerow([program['program_name'], program['policy_url']])
```

**Comment by yesnet0** [comment]
Thanks @codingo!

**Comment by codingo** [comment]
Closed in #145
